doc/usecases/kdb/README.md

```diff
@@ -0,0 +1,7 @@
+# Use cases for KDB
+
+This folder contains the use cases for the Key Database (KDB).
+These primarily are the basis for `libelektra-kdb`, but may also influence other things like, e.g., the high-level API `libelektra-highlevel`.
```

Comment:
Obvious to you and me, but not to somebody new, who may start with reading use cases. In general it would also be possible to have a completely different API that isn't built on another existing (public) API.